src/layouts/components/Homepage/HeroBody.tsx

```diff
@@ -179,13 +177,17 @@ const HeroSideSectionLayout = ({
   errors,
   title,
   subtitle,
-  size = "half",
+  size = "50%",
   alignment = "left",
-  backgroundColor = "black",
 }: HeroSideSectionProps) => {
-  const [, setSectionSize] = useState(size)
-  const [, setSectionAlignment] = useState(alignment)
-  const [, setSectionBackgroundColor] = useState(backgroundColor)
+  const { onChange } = useEditableContext()
+  const onClick = (value: SectionBackgroundColor) =>
+    onChange({
+      target: {
+        id: "section-0-hero-backgroundColor",
+        value,
+      },
+    })
```

Comment:
Nit: could we use `onClickColor`? This was a little confusing because there are multiple areas which involve a click to change, but this is only being used by the colour buttons

Reply:
not sure what you mean here - if you want me to extract this to a function called `onClickColor`, it's fine by me!

Reply:
updated to `onIconButtonClick`

Reply:
Sorry i mean to just rename the method for clarity!